src/routes/jira/configuration/jira-configuration-get.test.ts

```diff
@@ -64,15 +70,16 @@ describe("Jira Configuration Suite", () => {
 
 	it("should return success message after page is rendered", async () => {
 		const response = mockResponse();
+		mocked(booleanFlag).mockResolvedValue(true);
```

Comment:
just an FYI, this means that all boolean flags are true everywhere instead of the specific one you want.  If you look at the other tests, we have a `when` convention so you can specify the FF you want to return true which will remove all potential side effects from this test

Reply:
Used `when` to mock only one FF.